src/dataServices/dynamoDbDataService.test.ts

```diff
@@ -86,3 +96,173 @@ describe('updateResource', () => {
         expect(serviceResponse.resource).toMatchObject(expectedResource);
     });
 });
+
+describe('initiateExport', () => {
+    const initiateExportRequest: InitiateExportRequest = {
+        requesterUserId: 'userId-1',
+        exportType: 'system',
+        transactionTime: '2020-09-01T12:00:00Z',
+        outputFormat: 'ndjson',
+        since: '2020-08-01T12:00:00Z',
+        type: 'Patient',
+        groupId: '1',
+    };
+
+    test('Successful initiate export request', async () => {
+        // BUILD
+        // Return an export request that is in-progress
+        AWSMock.mock('DynamoDB', 'query', (params: QueryInput, callback: Function) => {
+            callback(null, {
+                // requesterUserId does not equal requesterUserId from initiateExportRequest therefore we are below throttle limit
+                Items: [DynamoDBConverter.marshall({ requesterUserId: 'userId-2', jobStatus: 'in-progress' })],
+            });
+        });
+        AWSMock.mock('DynamoDB', 'putItem', (params: QueryInput, callback: Function) => {
+            // Successfully update export-request table with request
+            callback(null, {});
+        });
+
+        const dynamoDbDataService = new DynamoDbDataService(new AWS.DynamoDB());
+
+        // OPERATE
+        const jobId = await dynamoDbDataService.initiateExport(initiateExportRequest);
+
+        // CHECK
+        expect(jobId).not.toBeUndefined();

```

Comment:
whew double negatives are hard could we just use:
```suggestion
        expect(jobId).toBeDefined();

```

Reply:
updated!